spec/views/user_index_spec.rb

```diff
@@ -0,0 +1,35 @@
+# _string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Login', type: :system do
+  describe 'User' do
+    before(:each) do
+      @user1 = User.create! name: 'Chaw', password: '123456', email: 'chaw@gmail.com', confirmed_at: Time.now
+      visit root_path
+      fill_in 'Email', with: 'chaw@gmail.com'
+      fill_in 'Password', with: '123456'
+      click_button 'Log in'
+      visit root_path
+    end
+
+    it 'shows the username' do
+      expect(page).to have_content('Chaw')
+    end
+
```

Comment:
- [ ] Since this test is to ascertain that you can see the username of __all other users__, and not the username of the __user__, kindly add one or more users and test the visibility of their usernames

Reply:
Thank you, I have added one more user and tested the visibility of their usernames